docs/source/international_languages.md

```diff
@@ -0,0 +1,232 @@
+organize and share the work to translate, maintain, and manage a language
+repository. Working with other contributors and maintainers on a language
+translation can be a very satisfying activity for you and other Fabric users.
+
+Follow these instructions to create your own language repository. Our
+instructions will use the Sanskrit language as an example:
+
+1. Identify the [ISO
+   639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) two letter
+   international language code for your language. Sanskrit has the two letter
+   code `sa`.
+
+1. Clone the main Fabric repository to your local machine, renaming the
+   repository during the clone:
+   ```bash
+   git clone git@github.com:hyperledger/fabric.git fabric-docs-sa
+   ```
+
+1. Select the Fabric version you are going to use as a baseline. We
+   recommend that you start with at least Fabric 2.1, and ideally a Long Term
+   Support version such as 2.2. You can add other releases later.
+
+   ```bash
+   cd fabric-docs-sa
+   git fetch origin
+   git checkout release-2.1
+   ```
+
+1. Remove all folders from the root directory except `/doc`. Likewise, remove
+   all files (including hidden ones) except `LICENCE` and `README.md`, so that
+   you are left with the following:
+
+   ```bash
+   ls -l
+   total 40
+   -rw-r--r--   1 user  staff  11358  5 Jun 14:38 LICENSE
+   -rw-r--r--   1 user  staff   4822  5 Jun 15:09 README.md
+   drwxr-xr-x  11 user  staff    352  5 Jun 14:38 docs
+   ```
+
+1. Update the `README.md` file to using [this
+   one](https://github.com/hyperledger-labs/fabric-docs-ml/blob/master/README.md)
+   as an example.
+
+   Customize the `README.md` file for your new language.
+
+1. Add a `.readthedocs.yml` file [like this
+   one](https://github.com/hyperledger-labs/fabric-docs-ml/blob/master/.readthedocs.yml)
+   to the top level folder.  This file is configured to disable ReadTheDocs PDF
+   builds which may fail if you use non-latin character sets. Your top level
+   repository folder will now look like this:
+
+   ```bash
+   (base) anthonyodowd/git/fabric-docs-sa ls -al
+   total 96
+   ...
+   -rw-r--r--   1 anthonyodowd  staff    574  5 Jun 15:49 .readthedocs.yml
+   -rw-r--r--   1 anthonyodowd  staff  11358  5 Jun 14:38 LICENSE
+   -rw-r--r--   1 anthonyodowd  staff   4822  5 Jun 15:09 README.md
+   drwxr-xr-x  11 anthonyodowd  staff    352  5 Jun 14:38 docs
+   ```
+
+1. Commit these changes locally to your local repository:
+
+   ```bash
+   git add .
+   git commit -s -m "Initial commit"
+   ```
+
+1. Create a new repository on your GitHub account called `fabric-docs-sa`. In
+   the description field type `Hyperledger Fabric documentation in Sanskrit
+   language`.
+
+1. Update your local git `origin` to point to this repository, replacing
+   `YOURGITHUBID` with your GitHub ID:
+
+   ```bash
+   git remote set-url origin git@github.com:YOURGITHUBID/fabric-docs-sa.git
+   ```
```

Comment:
I think we should tell them at this point to run the command 
git remote -v and they should see both the `upstream` and `origin` targets. 
This will allow them to push changes to the repo while pulling in changes from other translators that they are collaborating with.

Reply:
Good idea, I'll add that, and it gives a nice confirmatory output.  Thx

Reply:
I should add that the user cannot create an upstream yet, until the repository has been created in HL Labs.  I will add a comment to this effect. Thx